app/serializers/project_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSerializer < ActiveModel::Serializer
```

Comment:
Hey @alexpeachey that definitely makes sense regarding versioning. I might be misunderstanding, but I believe those docs refer to the namespacing of models and serializers, rather than controllers. In the tracker project, the controllers are namespaced, but the models aren't. So it was looking for the serializers in `app/serializers`, but not finding them and instead rendering the default json- that's why we get `updated\created_at` back in the `show` endpoints. That's what I wanted to check to see if it was an intentional design, but keeping the versioning and explicitly specifying the serializers in the other actions as you suggest seems like a better fix than just dropping the versioning.